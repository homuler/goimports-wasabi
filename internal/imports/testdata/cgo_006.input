// keep import C declarations
package main

// #include <stdio.h>
// #include <stdlib.h>
import "C"

import (
	"context"
	"fmt"
	"unsafe"
)

import (
	// static void myprint(char* s) {
	//   printf("%s\n", s);
	// }
	"C"
)

func main() {
	cs := C.CString("Hello from stdio")
	C.myprint(cs)
	C.free(unsafe.Pointer(cs))
	fmt.Println(context.Background())
}
