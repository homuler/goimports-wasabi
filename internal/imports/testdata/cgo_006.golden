// keep import C declarations
package main

// #include <stdio.h>
// #include <stdlib.h>
import "C"

import (
	//	static void myprint(char* s) {
	//	  printf("%s\n", s);
	//	}
	"C"
)

import (
	"context"
	"fmt"
	"unsafe"
)

func main() {
	cs := C.CString("Hello from stdio")
	C.myprint(cs)
	C.free(unsafe.Pointer(cs))
	fmt.Println(context.Background())
}
